core/src/main/java/com/amplifyframework/api/GraphQlBehavior.java

```diff
@@ -371,14 +395,16 @@
      * The subscription may be canceled by calling
      * {@link GraphQLOperation#cancel()}.
      * Requires that only one API is configured in your
-     * `amplifyconfigurationl.json`. Otherwise, throws an IllegalStateException.
+     * `amplifyconfigurationl.json`. Otherwise, emits an ApiException to
```

Comment:
amplifyconfiguration.json (no l)

Reply:
A bunch of the others have this typo as well since it was probably copy/pasted

Reply:
good catch. fixed